Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN apt-get update && apt-get install -y \
     pkg-config \
     python \
     xz-utils \
+    zip \
```

Comment:
If all OS have 7z command line utility available, we may do `7z -tzip -mx=9 a something.zip somefile` instead, for better (but still zip) compression.

If wanting to be more pedantic, `advzip -z -4 -k something.zip` can recompress an existing zip using zoplfi and shave more bytes.

Reply:
I use the Docker builds for everyday development, running them each time I change the code. So I was looking for a compromise here - something which doesn't take too long, but is good enough to use in the release.

Maybe there could be a way to make an option for a Docker script which specializes it for a final release, but I don't know how to do it. I think it's very bad to add extra manual steps to the release process - this results in mistakes.

Reply:
If we can do 7z everytime (maybe it's provided by msys2 and brew?), this would be a very good compromise. 7z level 9 zip compression is fast enough, while doing 99.999999% of what can provide insane minute-long zopfli.

I agree extra manual steps must be avoided when possible.

Reply:
I found the way to add an optional step to the Docker script which is only run by passing an extra arg `--build-arg=release=1`. So this let me add a slower zip step using 7z.

Reply:
If doing a “release build” option, maybe use `7z` for both, so the non-release build would validate the same software the release build uses? For example by doing `7z -tzip -mx=9 a archive.zip files` for release, and `7z -tzip a archive.zip files` for usual builds (or even `7z -tzip -mx=0 a archive.zip files` for blazing fast zip archiving (no compression). I'm a bit concerned if the daily build does not test the release process. We can just tune the compression level while using the same tool.